.github/workflows/release-please.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          submodules: recursive
```

Comment:
We don't need the submodules?

Reply:
No, it was replaced with [this](https://github.com/open-feature/flagd/blob/main/go.mod#L15).

Reply:
Thanks 👍